db/migrate/20200630141828_update_application_status_to_offer_withdrawn.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateApplicationStatusToOfferWithdrawn < ActiveRecord::Migration[6.0]
+  def change
+    ApplicationChoice
+      .where(status: 'rejected')
```

Comment:
This is not watertight as it's possible for an offer to have transitioned from offer to withdrawal to offer to rejection

Reply:
Is that true? Offer to rejection isn't possible, just offer to offer withdrawn.

Otherwise it's an existing bug in the scope & attribute:

https://github.com/DFE-Digital/apply-for-teacher-training/blob/2ebbc1127650f277fb6795228ed70d95bd81206b/app/models/application_choice.rb#L34-L38

Reply:
Yes, you're right!